migrations/2025-01-07-105739_create_index_for_relay/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+CREATE UNIQUE INDEX relay_profile_id_connector_reference_id ON relay (profile_id, connector_reference_id);
```

Comment:
Nit:
```suggestion
CREATE UNIQUE INDEX relay_profile_id_connector_reference_id_index ON relay (profile_id, connector_reference_id);
```

Update the `down.sql` file as well if you're taking up this change.